tests/installation/bootloader_s390.pm

```diff
@@ -139,40 +140,18 @@ EO_frickin_boot_parms
             # ensure that we are in cms mode before executing qaboot
             $s3270->sequence_3270("String(\"#cp i cms\")", "ENTER", "ENTER", "ENTER", "ENTER",);
             $r = $s3270->expect_3270(output_delim => qr/CMS/, timeout => 20);
-            $s3270->sequence_3270("String(\"qaboot $repo_host $dir_with_suse_ins\")", "ENTER", "Wait(InputField)",);
-            # wait for qaboot dumping us into xedit. If this fails, probably the
-            # download of kernel or initrd timed out and we retry
-            $r = $s3270->expect_3270(buffer_ready => qr/X E D I T/, timeout => 60);
+            $s3270->sequence_3270("String(\"getfiles $zvm_guest $repo_host $dir_with_suse_ins/boot/s390x\")", "ENTER");
+            $r = $s3270->expect_3270(output_delim => qr/Opening BINARY mode data connection for linux/, timeout => 20);
+            $r = $s3270->expect_3270(output_delim => qr/Opening BINARY mode data connection for initrd/, timeout => 20);
+            $r = $s3270->expect_3270(output_delim => qr/bytes transferred in/, timeout => 20);
         };
         last unless ($@);
         diag "s3270 sequence failed: $@";
         diag "Maybe the network is busy. Retry: $_ of $max_retries";
     }
     die "Download of Kernel or Initrd took too long (with retries)" unless $r;
 
-    $s3270->sequence_3270(qw( String(INPUT) ENTER ));
-
-    $r = $s3270->expect_3270(buffer_ready => qr/Input-mode/);
-
-    # can't use qw() because of space in commands...
-    $s3270->sequence_3270(split /\n/, $sequence);
-
-    $r = $s3270->expect_3270(buffer_ready => qr/X E D I T/);
-
-    ## Remove the "manual=1" and the empty line at the end
-    ## of the parmfile.
-
-    ## HACK HACK HACK HACK this code just 'knows' there is
-    ## an empty line and a single "manual=1" at the bottom
-    ## of the ftpboot parmfile.  This may fail in obscure
-    ## ways when that changes.
-    $s3270->sequence_3270(qw(String(BOTTOM) ENTER String(DELETE) ENTER));
-    $s3270->sequence_3270(qw(String(BOTTOM) ENTER String(DELETE) ENTER));
-
-    $r = $s3270->expect_3270(buffer_ready => qr/X E D I T/);
-
-    # save the parmfile.  ftpboot then starts the installation.
-    $s3270->sequence_3270(qw( String(FILE) ENTER ));
+    $s3270->sequence_3270("String(\"install LINUX222\")", "ENTER");
```

Comment:
Can you remove the hardcode of LINUX222 in favor of using a parameter or variable?

Reply:
Sure, I just forgot it